### DISCUSSION TITLE
Oops，这里是 404 页面 504错误

### DISCUSSION THREAD

**Discussion by Destroy-God** [spencerwooo/onedrive-vercel-index#724] [Q&A] [answered]
![屏幕截图 2022-07-25 214940](https://user-images.githubusercontent.com/67398572/180793281-d8b543d3-1fde-446d-a978-98c69d4042fe.png)
E5订阅，不知道怎么回事，baseDirectory填对了，Redis也有，但就是不行，一直504
![屏幕截图 2022-07-25 214754](https://user-images.githubusercontent.com/67398572/180793163-59a60c87-f3e9-4543-8ccf-0ff8d6be1855.png)
![屏幕截图 2022-07-25 214804](https://user-images.githubusercontent.com/67398572/180793174-8b4138cd-5e46-434c-93bd-807cc9aa7e9c.png)
![屏幕截图 2022-07-25 214849](https://user-images.githubusercontent.com/67398572/180793180-cef8f3c8-885a-4d22-bd06-5f0e18359ddc.png)
![屏幕截图 2022-07-25 214719](https://user-images.githubusercontent.com/67398572/180793192-ffb0feeb-91e2-4f41-b2ac-6f7b21306558.png)

**Comment by minterjia** [comment]
大佬解决了吗，我和你一样的错误

**Reply by starlight02** [reply]
请尝试 https://github.com/spencerwooo/onedrive-vercel-index/discussions/722#discussioncomment-3423379

**Comment by starlight02** [comment]
请尝试 https://github.com/spencerwooo/onedrive-vercel-index/discussions/722#discussioncomment-3423379

**Reply by Destroy-God** [reply]
对不起，我没改过密码

**Comment by Destroy-God** [comment, marked as answer]
问题已经解决了，解决办法是使用自己的 client id 与 secret

**Reply by myh1990** [reply]
请教下 client id 与 secret在哪里获取，有教程链接不
